### DISCUSSION TITLE
[BUG] Cold start of Firefox sometimes prompts for troubleshoot mode

### DISCUSSION THREAD

**Discussion by metametapod** [rxhanson/Charmstone-Community#50] [General]
# Description

When opening Firefox with Charmstone, there is occasionally a dialog prompting to open in troubleshoot mode. This does not occur when launching Firefox via the Dock or Finder. To reproduce:

1. Set Firefox to near shortcut (see first screenshot below).
2. Quit Firefox via **Firefox** > **Quit Firefox** in menubar or Command-Q shortcut.
3. Open Firefox via Charmstone with near shortcut.
4. Repeat steps 2-4 until dialog in second screenshot is shown (intermittent).

# Screenshots

<img width="688" alt="Screenshot 2024-07-30 at 16 16 48" src="https://github.com/user-attachments/assets/60dc72d4-db6e-4f54-a9f2-d4f6455f1d0f">

<img width="512" alt="Screenshot 2024-07-30 at 16 12 52" src="https://github.com/user-attachments/assets/1b5e73b9-0ab8-4205-a0b9-9d2a647332e4">

**Comment by rxhanson** [comment]
Troubleshoot mode is enabled in Firefox if you hold the option key while launching it, and I don't think there's a way to disable that behavior? As a workaround, you could go with a different set of modifier keys.

https://support.mozilla.org/en-US/kb/diagnose-firefox-issues-using-troubleshoot-mode#w_how-to-start-firefox-in-troubleshoot-mode

**Comment by metametapod** [comment]
Thanks, that was the issue. Using different modifier keys fixed it.
